Kernel/FileSystem/FileBackedFileSystem.h

```diff
@@ -31,33 +31,42 @@
 
 namespace Kernel {
 
-class DiskBackedFS : public FS {
+class FileBackedFS : public FS {
 public:
-    virtual ~DiskBackedFS() override;
+    virtual ~FileBackedFS() override;
 
-    virtual bool is_disk_backed() const override { return true; }
+    virtual bool is_file_backed() const override { return true; }
 
-    BlockDevice& device() { return *m_device; }
-    const BlockDevice& device() const { return *m_device; }
+    File& file() { return *m_file; }
+    FileDescription& file_description() { return *m_file_description; }
+    const File& file() const { return *m_file; }
+    const FileDescription& file_description() const { return *m_file_description; }
```

Comment:
Can't we get the file from the file description, instead of storing it explicitly?